fission-cli/library/Fission/CLI/IPFS/Version/Types.hs

```diff
@@ -0,0 +1,26 @@
+module Fission.CLI.IPFS.Version.Types (Version (..)) where
+
+import           Servant.API
+
+import           Fission.Prelude
+
+data Version = Version
+  { major :: Word8
+  , minor :: Word8
+  , patch :: Word8
+  }
+  deriving (Show, Eq)
+
+instance Display Version where
+  display Version {..} =
+    mconcat
+      [ "v"
+      , display @Int $ fromIntegral major
+      , "."
+      , display @Int $ fromIntegral minor
+      , "."
+      , display @Int $ fromIntegral patch
+      ]
```

Comment:
We want this printed in a very specific format, so this is a lightweight way to enforce this to be correct. Also may be reusable.